php/libraries/ExternalIDGenerator.php

```diff
@@ -27,15 +29,15 @@ class ExternalIDGenerator extends SiteIDGenerator
     /**
      * Generates ExternalIDs.
      *
-     * @param string $siteAlias    To be appended to the ID value. Usually an
-     *                             abbreviation for the name of a site.
-     * @param string $projectAlias To be appended to the ID value. Usually an
-     *                             abbreviation for the name of a project.
+     * @param ?SiteAlias    $siteAlias    The site prefix to prepend to the ID value.
+     * @param ?ProjectAlias $projectAlias To be appended to the ID value.
```

Comment:
```suggestion
     * @param ?ProjectAlias $projectAlias The project prefix to prepend to the ID value.
```
To be consistent (see php/libraries/PSCIDGenerator.php lines 33-36).

Reply:
same here